src/js/lib/cookies.js

```diff
@@ -0,0 +1,15 @@
+export const Cookies = {
+	get: () => document.cookie,
+}
+
+/**
+ * Get the value of a particular cookie
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie
+ * @param {string} key Which cookie value to get
+ * @return {string | undefined} The value of the specified key, or `undefined` if it's not set.
+ */
+export const getCookieValue = ( key ) => Cookies.get()
```

Comment:
How "namespaced" is this constant? Does it add a `getCookieValue` to the global space that could clash with other plugins or core?

Reply:
Everything in our built scripts is scope-limited to the module.  Nothing will pollute the global scope unless it's attached to `global` or `window`, etc.